.codespell/ignore-words.txt

```diff
@@ -1,19 +1,27 @@
 adaptee
 ancilliary
 ans
+anull
```

Comment:
Just curious, why do we need this? `git grep` didn't find this word

Reply:
Mixed caps :)

```
peps/pep-0543.rst:838: aNULL ==> annul
```
```console
❯ git grep anull

❯ git grep -i anull
peps/pep-0543.rst:    'ECDH+AESGCM:ECDH+CHACHA20:DH+AESGCM:DH+CHACHA20:ECDH+AES256:DH+AES256:ECDH+AES128:DH+AES:RSA+AESGCM:RSA+AES:!aNULL:!eNULL:!MD5'
```

Reply:
Ah, didn't think of that!